### ISSUE TITLE
Form owner that is not an AW administrator cannot cancel form

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/HPI#816]
The cancel form action has the `condition-isuserformowneroradministrator` condition configured.  This should allow the user to cancel the form if the user is the form owner or in the `wizard_administrators` group.  On edge2, I created a form as a user that is not an administrator.  However I cannot cancel the form while it is in the Draft state.  I know that the user owns the form because I see it in the 'My Forms` dashlet:

**This user owns this form**
![image](https://cloud.githubusercontent.com/assets/1377228/9888029/600b7db2-5bb8-11e5-9272-806004315565.png)

**Cancel Form in Why Can't I**
![image](https://cloud.githubusercontent.com/assets/1377228/9888018/52b2a938-5bb8-11e5-9c2c-fcd468486e67.png)

**Comment by gsteimer** [comment]
@renesaladrigas - I was able to reproduce this on edge2 today.  Here's what I'm seeing

1. If the form was created by someone in `wizard_administrators`, the close function works fine.  The form owner can close the form, or any other administrator.  This is why I couldn't reproduce the problem when we talked yesterday
1. BUT - if the form is created by someone who is only in `wizard_contributors` (not an administrator) - then only users in the wizard admin group can cancel the form - not the owner.

I tried it out on edge2 and created a form as `qdauthor1`.  What should happen is that `qdauthor1` **or anyone in `wizard_administrators` should be able to cancel the form**.

Instead, a wizard admin user can cancel the form, but while logged in as `qdauthor1`, I cannot cancel the form.  The `condition-isuserformowneroradministrator` condition should not be failing in this case because I'm logged in as the form owner.
